src/e2e-test/java/io/cdap/cdap/ui/utils/HttpRequestHandler.java

```diff
@@ -47,18 +45,15 @@ public static HttpResponse makeHttpRequest(HttpMethod method, String urlStr,
       : null;
     HttpRequest request = new HttpRequest(method, new URL(urlStr), reqHeaders, reqBody, bodyLength);
     HttpResponse httpResponse = HttpRequests.execute(request, new DefaultHttpRequestConfig(false));
-    if (httpResponse.getResponseCode() != HttpURLConnection.HTTP_OK) {
```

Comment:
I realized we'd better send the response regardless of success/failure. We might have tests that we'd look for a response code other than 200 (e.g., 404).